app/assets/stylesheets/application.css.scss

```diff
@@ -17,10 +17,13 @@
  */
 
  body {
-  background: {
-       image: asset-url("concert.jpg");
-     }
+  background-color: #eeeeee;
```

Comment:
Color literals like `#eeeeee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background-attachment, background-color, background-repeat, background-size